pipelines/azure-pipelines.yml

```diff
@@ -45,7 +45,7 @@ resources:
       name: defra/epr-webapps-code-deploy-templates
       type: github
       endpoint: defra
-      ref: main
+      ref: upgrade-net-8
```

Comment:
This needs to point back to main and an additional variable needs to be added under the 'variables' section called 'dotnetVersion' with the value of 'dotnetVersion8'. This then needs to be referred to under the 'parameters' section. I.e. dotnetVersion: ${{ variables.dotnetVersion}}

Reply:
Done